README.md

```diff
@@ -1,21 +1,27 @@
-# ElixirJanusPluginVideoroom
+# Elixir Janus Plugin VideoRoom
 
-**TODO: Add description**
+This package implements functionality to communicate with [Janus VideoRoom plugin](https://janus.conf.meetecho.com/docs/videoroom.html).
+
+It takes advantage of transport layer provided by another package [Elixir Janus](https://github.com/software-mansion-labs/elixir-janus).
+
+## Disclaimer
+Package is experimental and is not yet released to hex.
 
-## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `elixir_janus_plugin_videoroom` to your list of dependencies in `mix.exs`:
+## Installation
```

Comment:
Maybe some usage example? Even a tiny one (e.g sending one request and link to other messages that can be sent. If that would be too much work, at least put at TODO)